components/programs/ClassInstanceData.tsx

```diff
@@ -333,34 +419,89 @@ export const SummerDates: ClassDates = {
   demoDay: Date.parse("2020-07-25T13:00:00-07:00"),
 };
 
-function summerClass(
-  course: Course,
-  campus: Campus,
-  formAssemblyId: string
-): ClassInstance {
+export const FallDates: ClassDates = {
+  registrationDeadline: Date.parse("2020-08-23T20:00:00-07:00"),
+  interview: Date.parse("2020-09-02T12:00:00-07:00"),
+  demoDay: Date.parse("2020-12-12T13:00:00-07:00"),
+};
+
+// function summerClass(
+//   course: Course,
+//   campus: Campus,
+//   formAssemblyId: string
+// ): ClassInstance {
+//   return {
+//     type: "class",
+//     course,
+//     campus,
+//     classDates: SummerDates,
+//     meets: (
+//       <>
+//         Monday, Wednesday, Friday{" "}
+//         <Box component="span" display="inline-block">
+//           10:30am - 2:30pm
+//         </Box>
+//       </>
+//     ),
+//     startDate: "June 15th",
+//     endDate: "July 25th",
+//     signupUrl: `https://www.tfaforms.com/4804494?tfa_2013=${formAssemblyId}`,
+//   };
+// }
+
+function fallClassMW(course: Course, campus: Campus): ClassInstance {
+  return {
+    type: "class",
+    course,
+    campus,
+    classDates: FallDates,
+    meets: (
+      <>
+        Monday, Wednesday{" "}
+        <Box component="span" display="inline-block">
+          4:00pm - 6:30pm
+        </Box>
+      </>
+    ),
+    startDate: "September 14th",
+    endDate: "December 12th",
+    signupUrl: `https://www.tfaforms.com/4840819`,
+  };
+}
+
+function fallClassTH(course: Course, campus: Campus): ClassInstance {
   return {
     type: "class",
     course,
     campus,
-    classDates: SummerDates,
+    classDates: FallDates,
     meets: (
       <>
-        Monday, Wednesday, Friday{" "}
+        Tuesday, Thursday{" "}
         <Box component="span" display="inline-block">
-          10:30am - 2:30pm
+          4:00pm - 6:30pm
         </Box>
       </>
     ),
-    startDate: "June 15th",
-    endDate: "July 25th",
-    signupUrl: `https://www.tfaforms.com/4804494?tfa_2013=${formAssemblyId}`,
+    startDate: "September 14th",
+    endDate: "December 12th",
+    signupUrl: `https://www.tfaforms.com/4840819`,
   };
 }
 
-export const SummerClassInstances: ClassOrWorkshopInstance[] = [
-  summerClass(Courses.web_bootcamp, Campuses.online_sf, "tfa_2247"),
-  summerClass(Courses.game_bootcamp, Campuses.online, "tfa_2248"),
-  summerClass(Courses.web_bootcamp, Campuses.online_oakland, "tfa_2245"),
+// export const SummerClassInstances: ClassOrWorkshopInstance[] = [
+//   summerClass(Courses.web_bootcamp, Campuses.online_sf, "tfa_2247"),
+//   summerClass(Courses.game_bootcamp, Campuses.online, "tfa_2248"),
+//   summerClass(Courses.web_bootcamp, Campuses.online_oakland, "tfa_2245"),
+// ];
+
+export const FallClassInstances: ClassOrWorkshopInstance[] = [
+  fallClassMW(Courses.web_class, Campuses.online_sf),
```

Comment:
This should probably be `Campuses.online` like all of the others, the layout doesn't work very well with as many skills as this course has with the longer course name